Responder.js

```diff
@@ -59,7 +59,7 @@ module.exports = class {
 		if (message.key.fromMe) return
 
 		const senderID = message.key.remoteJid
-		const messageText = message.message?.conversation || message.message?.extendedTextMessage?.text
```

Comment:
If message is null, won't the code break?

Reply:
I didn't catch any execution where the message came null yet, but we can protect the block with "ifs", or use early return (more elegant)

Reply:
Yeah, please add in the checks. Because there are messages with no `message` content. For example, messages where you're added to a group